platform/viewer/src/utils/initWebWorkers.test.js

```diff
@@ -20,4 +20,4 @@ describe('initWebWorkers', () => {
       cornerstoneWADOImageLoader.webWorkerManager.initialize
     ).toHaveBeenCalledTimes(1);
   });
-});
+}); */
```

Comment:
Hmm.. I guess we should/would mock `cornerstoneWADOImageLoader` here

Reply:
The test it self got removed as part of clean-up.